### ISSUE TITLE
Use PJ everywhere (JSON)

### ISSUE THREAD

**Issue by Maijin** [rizinorg/rizin#200]
**Is your feature request related to a problem? Please describe.**

There are at least 4 JSON implementations in rizin source.

**Describe the solution you'd like**

Keep only pj.c and kill the others:

- [x] dsojson
- [x] handmade printf json
- [ ] sdbjson

**Comment by ret2libc** [comment]
As it was noticed on Mattermost, `RJson` and `pj` do different things, so `RJson` should be kept at least, together with `pj`.
`pj` is for formatting the output, `RJson` for parsing JSON.

**Comment by eagleoflqj** [comment]
I'll take care of handmade printf json on upstream. Please sync, focus on other stuff and avoid duplicate labor.

**Comment by Maijin** [comment]
> RJson and pj do different things, so RJson should be kept at least, together with pj.

Right, I forgot, then they should be merged I think 😄

**Comment by XVilka** [comment]
Here is what brief search shows (after https://github.com/rizinorg/rizin/pull/429 is merged):
```
[i] ℤ rg "printf[ ]*\(\"\{" librz/                                                                                                                                                                                                       librz/core/visual.c
3775:                           rz_cons_printf("{%s}", word);

librz/util/format.c
2312:                                   p->cb_printf("{\"name\":\"%s\",\"type\":\"", fieldname);
2314:                                   p->cb_printf("{\"type\":\"");

librz/config/config.c
78:                     cfg->cb_printf("{");
249:                    cfg->cb_printf("{");

librz/bin/format/xnu/rz_cf_dict.c
373:    printf("{");
```

**Comment by XVilka** [comment]
@Maijin is there anything left? I think all manually crafted JSON and dsojson were removed/switched to the existing APIs.

**Comment by Maijin** [comment]
Yes but we can open a new issue - as I explained I think we should extract sdbjson, pj and R_Json into a lib. 
And have it under one unique code for both creating and reading JSON.

**Comment by XVilka** [comment]
The only user of manual JSON creation now is `librz/util/format.c` (`pf` handler).

**Comment by XVilka** [comment]
Also blocked by https://github.com/rizinorg/rizin/issues/985

**Comment by ret2libc** [comment]
@XVilka @Maijin what's the status of this?

**Comment by Maijin** [comment]
See the blocked by issue referenced by Xvilka

**Comment by XVilka** [comment]
Closed in favor of https://github.com/rizinorg/rizin/issues/985

**Comment by ret2libc** [comment]
@XVilka how are #905 and #783 related to PJ?

**Comment by XVilka** [comment]
Ah, my bad, `pf` was converted to the PJ as well.
